src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -1503,6 +1506,27 @@ private PBXNativeTarget generateBinaryTarget(
     return target;
   }
 
+  private ImmutableList<String> getExcludesForModulesUnderTests(
+      TargetNode<? extends CxxLibraryDescription.CommonArg, ?> targetNode) {
```

Comment:
Nit: Let's clarify that `targetNode` is actually the `testingTarget`.

Reply:
👍